### ISSUE TITLE
Likely an issue in Structure.as_dict() for abivars format

### ISSUE THREAD

**Issue by computron** [materialsproject/pymatgen#406]
Line 1322 of structure.py reads:

```
elif geomode == "angdeg":
                d.update(dict(
                    acell=3 * [1.0],
                    angdeg=angdeg))
```

But in the last line, it looks like angdeg is never defined (at least, it is underlined in red in my IDE). So if this code were to be run, it would likely choke.

This won't affect things unless the abivars format is specified for the dict and the geomode is "angdeg" (I have no idea what these mean)

**Comment by gmatteo** [comment]
Hi,

Abinit supports different set of variables for the specification of the crystal structure.
That section is supposed to handle the case in which the dictionary specifies the lattice in terms of 
angles and vector lengths.
There's obviously an issue when `geomode == "angdeg"`. I will fix it.
Thanks for reporting the problem.

Best,
Matteo

**Comment by shyuep** [comment]
Has this been fixed?

**Comment by shyuep** [comment]
@gmatteo Has this been fixed? Can we close this issue?

**Comment by gmatteo** [comment]
Yes, fixed in #720
